spi_nand_flash/src/nand.c

```diff
@@ -139,15 +139,21 @@ static esp_err_t spi_nand_micron_init(spi_nand_flash_device_t *dev)
         .miso_data = &device_id
     };
     spi_nand_execute_transaction(dev->config.device_handle, &t);
-    dev->read_page_delay_us = 115;
     dev->erase_block_delay_us = 2000;
-    dev->program_page_delay_us = 240;
     switch (device_id) {
     case MICRON_DI_34:
+        dev->read_page_delay_us = 115;
+        dev->program_page_delay_us = 240;
         dev->dhara_nand.num_blocks = 2048;
         dev->dhara_nand.log2_ppb = 6;        // 64 pages per block
         dev->dhara_nand.log2_page_size = 12; // 4096 bytes per page
         break;
+    case MICRON_DI_14:
+    case MICRON_DI_15:
```

Comment:
Do you know what is the difference between these two device IDs?

Reply:
It's operating voltage. The chip can run on two voltage levels: device ID 14 - 3.3V, device ID 15 -1.8V

Reply:
Given that ESP chips only have 3.3V IOs, should the device ID 15 be excluded?

Reply:
We could leave it out, but I don't see any harm in keeping it (some people may design parts of their circuits around 1.8V and decide to supply their storaage from that as well).